build/Build.cs

```diff
@@ -73,7 +73,7 @@ class Build : NukeBuild
                 .SetProjectFile(Solution)
                 .SetConfiguration(Configuration)
                 .SetLoggers("trx")
-                .SetVerbosity(DotNetVerbosity.Normal)
+                .SetVerbosity(DotNetVerbosity.normal)
```

Comment:
Nuke changelog [says this is fixed](https://github.com/nuke-build/nuke/blob/59bdf80fcbe9b11f9e443a701d455b5aaa58c532/CHANGELOG.md?plain=1#L21) in 8.0.0, but... it appears to need to be lowercase still. :shrug: